### DISCUSSION TITLE
Is it chiseled or chiselled?

### DISCUSSION THREAD

**Discussion by dalibormesaric** [dotnet/dotnet-docker#5027] [General]
It looks like microsoft uses chiseled https://devblogs.microsoft.com/dotnet/announcing-dotnet-chiseled-containers/ but canonical uses chiselled https://ubuntu.com/blog/combining-distroless-and-ubuntu-chiselled-containers ... why?

**Comment by lbussell** [comment]
"Chiseled" is the American English way to spell the word, they are both valid!
